contracts/test/TestUtil.sol

```diff
@@ -666,11 +679,24 @@ contract TestUtil is DSTest {
     }
 
     function mintFundsAndDepositIntoPool(LP lp, Pool pool, uint256 mintAmt, uint256 liquidityAmt) internal {
-        if (mintAmt > uint256(0)) mint("USDC", address(lp), mintAmt);
+        if (mintAmt > uint256(0)) {
+            mint("USDC", address(lp), mintAmt);
+        }
+
         lp.approve(USDC, address(pool), MAX_UINT);
+
         assertTrue(lp.try_deposit(address(pool), liquidityAmt)); 
     }
 
+    function noAssert_mintFundsAndDepositIntoPool(LP lp, Pool pool, uint256 mintAmt, uint256 liquidityAmt) internal {
+        if (mintAmt > uint256(0)) {
+            mint("USDC", address(lp), mintAmt);
+        }
+
+        lp.approve(USDC, address(pool), MAX_UINT);
+        lp.deposit(address(pool), liquidityAmt); 
+    }
```

Comment:
Lets just keep this function and rename it to `mintFundsAndDepositIntoPool` and delete the other one